### DISCUSSION TITLE
Is there automatic build testing after committing to a branch?

### DISCUSSION THREAD

**Discussion by pcw24601** [GSTT-CSC/hazen#115] [Q&A] [answered]
On Bitbucket builds were automatically tested--does this happen in Github? If so, how do you view it?

**Comment by hshuaib90** [comment]
This needs to be set up as Github Actions on the repository, unfortunately, CI/CD doesn't translate well between systems. Unless you have an interest and the time to do it, I'll try to set it up sometime next week.

**Reply by pcw24601** [reply]
Thanks Haris. This would be a steep learning curve for me to set up, and I can't commit to that in the foreseeable future. It's not a major issue as I check all the tests locally, but on a couple of occasions the Bitbucket testing failed where my local testing passed, sometimes an issue with developing on Windows and testing on a different platform.

**Reply by hshuaib90** [reply]
No worries! I'll try to set it up asap, I'll also see what we can do about setting up tests for Windows platform.

**Comment by hshuaib90** [comment, marked as answer]
@pcw24601  this is now fixed and merged into develop branch thanks to @laurencejackson !
